src/test/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImplTest.java

```diff
@@ -390,6 +394,66 @@ void processAudioForDownloadShouldReturnThreeSessionsWithDifferentNumbersOfAudio
         verify(audioOperationService, times(3)).trim(any(), any(), any(), any());
     }
 
+    @Test
+    void processAudioForDownloadShouldReturnTwoSessionsWhenProvidedAudiosWithVaryingStartEndTimes()
```

Comment:
Typo - should be `processAudioForPlaybackShould...` ?

Reply:
fixed typo